src/settings/FixedDueDateSchedule/components/EditSections/components/ScheduleCard/ScheduleCard.test.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  within,
+  fireEvent,
+} from '@testing-library/react';
+import moment from 'moment-timezone';
+
+import '../../../../../../../test/jest/__mock__';
+
+import { FormattedMessage } from 'react-intl';
+import {
+  Datepicker,
+  IconButton,
+} from '@folio/stripes/components';
+import { Field } from 'react-final-form';
+import { DATE_FORMAT } from '../../../../../../constants';
+import ScheduleCard, {
+  parseDate,
+} from './ScheduleCard';
```

Comment:
Please follow code style for `imports`:

- Imports from React
(including prop-types, react-intl and others) - imports from libraries
- **empty line** -
imports from folio
- **empty line** -
own components imports - own helpers/utils functions - own constants imports - own other imports
- **empty line** -
styles imports.

You can find code style rules [here](https://wiki.folio.org/display/FOLIJET/Code+style+rules+not+covered+by+eslint)

Reply:
it is not possible to follow this rules in tests because if we mock some component we have to import it after place where we mock that component